lib/components/SelectionCard/SelectionCard.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { useMediaQueries } from '../../main';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { TooltipContent } from '../Tooltip/TooltipContent';
+import { TooltipTrigger } from '../Tooltip/TooltipTrigger';
+
+export interface SelectionCardProps {
+  /** Card label */
+  label: string;
+  /** Icon content */
+  icon?: React.ReactNode;
+  /** Show the selection checkmark */
+  selected?: boolean;
+  /** Callback when the card is clicked */
+  onClick?: () => void;
+  /** Callback when the card is hovered/exited or focused/blurred */
+  setHovered?: (visible: boolean) => void;
+  /** Tooltip content that's displayed when the info button is pressed (mobile only) */
+  tooltipContent?: React.ReactNode;
+  /** Aria label for the info button */
+  infoAriaLabel?: string;
+}
+
+const CheckIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="41" height="41" viewBox="0 0 41 41" fill="none">
+    <circle cx="20.9126" cy="20.074" r="16" fill="#00C181" />
+    <path
+      d="M18.5794 27.7406L30.3294 15.9906L27.9961 13.6572L18.5794 23.0739L13.8294 18.3239L11.4961 20.6572L18.5794 27.7406Z"
+      fill="white"
+    />
+  </svg>
+);
+
+const InfoIcon = () => (
```

Comment:
Maybe we can use the `info` icon from Google Fonts for this.

E.g, not correct styles applied
```tsx
<span className="material-symbols-outlined size-24 sm:size-32 select-none" aria-hidden>
info
</span>

```

Reply:
I wasn't sure if which should be preferred, SVG's or Google Fonts. If I recall correctly, Sauli said that it would be nice if we had our own icons instead of Google Fonts, that's why I decided to copy the SVG from Figma. I did the same for the arrow icon in HeroCard.

I'll change this to use the font, maybe we'll discuss about SVG vs Font at later time. I'm usure the checkmark icon though, it's supposed to be a green circle with white checkmark in design, but with Google Font it's a single color icon. Maybe SVG is required in this case.

![image](https://github.com/user-attachments/assets/90a05670-c102-49e4-b0c5-c6834d9f9931)